### PR TITLE
Create gRPC client for interaction with the Rusk server

### LINKED ISSUE #344: Create a gRPC client for interaction with the Rusk server

### PR DESCRIPTION
**Pull request by julesdesmit**
Fixes #344

### CODE REVIEW

**Comment by julesdesmit** [comment]
This should be ready to review on its own now. There are still a couple things to work out on other ends right now though, the main items being Phoenix integration, and some updates to the protobuf structures.

File: pkg/rpc/client.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+// InitRuskClient opens the connection with the Rusk gRPC server, and
+// launches a goroutine which listens for RPCBus calls that concern
+// the Rusk server.
+//
+// As the Rusk server is a fundamental part of the node functionality,
+// this function will panic if the connection can not be established
+// successfully.
+func InitRuskClient(address string, rpcBus *rpcbus.RPCBus) {
+	conn, err := grpc.Dial(address, grpc.WithInsecure(), grpc.WithBlock())
```
Comment (ggoranov) [inline pkg/rpc/client.go]: This will block infinitely if Rust server is down/not responding. We can use dial timeout here and then panic?

Comment (julesdesmit) [reply pkg/rpc/client.go]: Good point

Comment (ggoranov) [inline pkg/rpc/client.go]: At closing the node or restarting the conn, we could need conn.Close()

**Comment by autholykos** [comment]
blocked by #380

**Pull request merged**